tests/console/validate_repos.pm

```diff
@@ -30,6 +31,12 @@ sub run {
                 Autorefresh => $repo->{autorefresh}
         });
     }
+    # Check whether unexpected repos existed.
```

Comment:
please don't add comments when it is obvious due to the message that write in the `die`.